L1Trigger/TrackFindingTracklet/interface/VMRouterCM.h

```diff
@@ -22,11 +22,18 @@ namespace trklet {
   class VMStubsTEMemory;
 
   struct VMStubsTEPHICM {
-    VMStubsTEPHICM(unsigned int seednumber_, std::vector<VMStubsTEMemory*> vmstubmem_)
-        : seednumber(seednumber_), vmstubmem(vmstubmem_) {}
-
-    unsigned int seednumber;                  //seed number [0,11]
-    std::vector<VMStubsTEMemory*> vmstubmem;  // m_vmstubmem[n] is the VMStubsTEMemory for the nth copy
+    VMStubsTEPHICM(unsigned int seednumber_,
+                   unsigned int stubposition_,
+                   std::vector<std::vector<VMStubsTEMemory*> > vmstubmem_)
+        : seednumber(seednumber_), stubposition(stubposition_), vmstubmem(vmstubmem_) {};
+
```

Comment:
The call `vmstubmem(vmstubmem_)` will force a copy of the already made copy (`vmstubmem_`). Better to do
``vmstubmem(std::move(vmstubmem_))` which allows move to happen. Alternatively, change the signature of the function to be `std::vector<std::vector<VMStubsTEMemory*> > const&`.

Reply:
@aehart & @jpearkes wrote this line of code, so may wish may wish to comment.